### PR TITLE
Add serviceLocalizedName for all services

### LINKED ISSUE #3943: Ensure remaining Services have correct offscreen text for Brand SVG
**Is your feature request related to a problem? Please describe.**

Currently only some of the services have the VisuallyHiddenText with the english part split with a `<span lang="en-GB"></span>`. All of the services should have this.

e.g.
`/thai` already has this
Screenshot /thai

`/tigrinya` does not
Screenshot /tigrinya

**Describe the solution you'd like**
The way to fix this is to update the Service configs.

Currently only some of the services have service config values of:

```
product
serviceLocalizedName
```
We should update all of the services so they have these fields. 
Note: `product` should include the english part, and `serviceLocalizedName` should include the localised name. e.g. for Tigrinya: 
```
product: 'BBC News',
serviceLocalizedName: 'ትግርኛ`
```
Note: you can find the text to use for these values in the field `brandName`. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Updating the cypress test to check that there is a lang="en-GB" on part of the visually hidden text would be useful. 
This implementation has already been tested with screenreaders, for other services. 

- [x] This feature is expected to need manual testing.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves #3943

**Overall change:** Split product name into product name and `serviceLocalizedName`

**Code changes:**

- Update config for all services (except News, Cymrufyw, Naidheachdan, and Scotland)
- Add E2E test for offscreen text

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/pages/testsForAllPages.js [hunk 1]
```diff
@@ -230,15 +230,34 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
     });
 
     describe('Header Tests', () => {
+      const localizedName =
```
Comment (sareh) [inline cypress/integration/pages/testsForAllPages.js]: It'd be good to rename this, to make it clear it's a boolean
```suggestion
      const hasLocalisedName =
```

File: cypress/integration/pages/testsForAllPages.js [hunk 2]
```diff
@@ -230,15 +230,34 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
...
-          appConfig[service][variant].serviceLocalizedName !== undefined
+          hasLocalisedName
             ? `${appConfig[service][variant].product}, ${appConfig[service][variant].serviceLocalizedName}`
             : appConfig[service][variant].product,
         );
       });
 
+      if (hasLocalisedName) {
+        it("should have offscreen text with product's language code set to English", () => {
+          cy.get('header a span span').should('have.attr', 'lang', 'en-GB');
```
Comment (sareh) [inline cypress/integration/pages/testsForAllPages.js]: It'd be good to be more specific, stating that we're looking at the visually hidden text within the Banner.

```suggestion
          cy.get('header div[class^="Banner"] span[class^="VisuallyHiddenText"] span').should('have.attr', 'lang', 'en-GB');
```

File: cypress/integration/pages/testsForAllPages.js [hunk 3]
```diff
@@ -230,15 +230,34 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
...
+        });
+
+        it('should not set the language code for localised name', () => {
+          cy.get('header a span')
+            .eq(0)
+            .should('not.have.attr', 'lang', 'en-GB');
+        });
+      } else {
+        it('should not have a language attribute if no serviceLocalizedName set', () => {
+          cy.get('header a span').should('not.have.attr', 'lang', 'en-GB');
```
Comment (sareh) [inline cypress/integration/pages/testsForAllPages.js]: Same here 
```suggestion
          cy.get('header div[class^="Banner"] span[class^="VisuallyHiddenText"]').should('not.have.attr', 'lang', 'en-GB');
```

**Review by AlistairGempf — APPROVED** [review]
This looks good to me.

**Review by Bopchy — APPROVED** [review]

File: src/app/lib/config/services/arabic.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ const service = {
     articleTimestampPrefix: 'جدّد في',
     atiAnalyticsAppName: 'news-arabic',
     atiAnalyticsProducerId: '5',
-    brandName: 'BBC News عربي',
+    brandName: 'BBC News',
     serviceLocalizedName: 'عربي',
```
Comment (DenisHdz) [inline src/app/lib/config/services/arabic.js:21]: Could you place the `product` property between the `brandName` and `serviceLocalizedName`, so we keep it consistent across files? :)

Comment (j-pendlebury) [reply src/app/lib/config/services/arabic.js:21]: good spot, fixing now 😄

**Review by DenisHdz — APPROVED** [review]
Looks good 👍

File: src/app/lib/config/services/arabic.js (hunk 4 above)
Comment (DenisHdz) [reply src/app/lib/config/services/arabic.js:21]: Yeah, it already existed. I meant to move it down between those two 😄

**Review by ryanmccombe — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Looks great. Thanks for doing this standardisation.

**Comment by j-pendlebury** [comment]
just waiting for all the E2E stuff to be fixed. No need adding more potential issues

**Pull request merged**